ci/ray_ci/windows_container.py

```
@@ -0,0 +1,38 @@
+import subprocess
+import sys
+from typing import List, Optional
+
+from ci.ray_ci.container import Container
+
+
+class WindowsContainer(Container):
+    def install_ray(self, build_type: Optional[str] = None) -> List[str]:
+        subprocess.check_call(
+            [
+                "docker",
+                "build",
+                "-t",
+                self._get_docker_image(),
```

`build_type` is not used? maybe throw an exception if it is not None?